EIPS/eip-191.md

```diff
@@ -55,29 +54,54 @@ Using `0x19` thus makes it possible to extend the scheme by defining a version `
 |    `0x01`    | [712][eip-712] | Structured data
 |    `0x45`    | [191][eip-191] | `personal_sign` messages
 
+#### Version `0x00`
+
+```
+0x19 <0x00> <intended validator address> <data to sign>
+```
+
+The version `0x00` has `<intended validator address>` for the version specific data. In the case of a Multisig wallet that perform an execution based on a passed signature, the validator address is the address of the Multisig itself. The data to sign could be any arbitrary data.
+
+#### Version `0x01`
+
+The version `0x01` is for structured data as defined in [EIP-712]
+
+#### Version `0x45` (E)
+
+```
+0x19 <0x45 (E)> <thereum Signed Message:\n" + len(message)> <data to sign>
+```
+
+The version `0x45` (E) has `<thereum Signed Message:\n" + len(message)>` for the version specific data. The data to sign could be any arbitrary data.
```

Comment:
So this treats the `E` as the version? I would explicitly mention that the `E` is consumed, and `thereum` is not a typo :P

Reply:
> So this treats the E as the version?

Exactly, it was confusing that's why i created this PR :)


I would add on the next line.
> NB: The **E** in `Ethereum Signed Message` refers to the version `0x45`. The character `E` is `0x45` in Hex which make the rest: `thereum Signed Message:\n + len(message)` the version specific data.

Reply:
```suggestion
The version `0x45` (E) has `<thereum Signed Message:\n" + len(message)>` for the version-specific data. The data to sign can be any arbitrary data.

> NB: The `E` in `Ethereum Signed Message` refers to the version byte 0x45. The character `E` is `0x45` in hexadecimal which makes the remainder, `thereum Signed Message:\n + len(message)`, the version-specific data.
```